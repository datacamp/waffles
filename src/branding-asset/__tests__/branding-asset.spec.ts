import * as brandingVariants from '../index';

describe('Branding Asset', () => {
  it('exports every Branding Asset asset', () => {
    expect(brandingVariants).toMatchSnapshot();
  });
});
