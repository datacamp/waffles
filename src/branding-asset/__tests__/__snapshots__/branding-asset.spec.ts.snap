// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branding Asset exports every Branding Asset asset 1`] = `
Object {
  "DataCampLogo": [Function],
  "DataCampLogoTag": [Function],
  "DataCampLogomark": [Function],
  "DataCampLogomarkTag": [Function],
}
`;
